syft/exceptions.py

```diff
@@ -174,9 +174,9 @@ def simplify(worker: "sy.workers.AbstractWorker", e):
         except AttributeError:
             attributes = {}
         return (
-            sy.serde._simplify(worker, tp.__name__),
-            sy.serde._simplify(worker, traceback_str),
-            sy.serde._simplify(worker, attributes),
+            sy.serde.msgpack.serde._simplify(worker, tp.__name__),
```

Comment:
I don't love these long chains of module names, but my thinking is that it makes clear just how far into the internals of another module this code is reaching.